jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -98,6 +98,7 @@ private static final class TidLock {
 	private String name;						// The Thread's name
 	private int priority = NORM_PRIORITY;			// The Thread's current priority
 	private boolean isDaemon;				// Tells if the Thread is a daemon thread or not.
+	private volatile int threadStatus;      // The Thread's state.
```

Comment:
the spacing after `;` looks inconsistent. a single space should be fine between `;` and the comment.

Reply:
fixed alignment, github view may still be off due to use of tabspace = 8 in the browser.

Reply:
fixed